packages/gestalt/dist/index.d.ts

```diff
@@ -1920,6 +1939,28 @@ interface TextFieldProps {
   value?: string | undefined;
 }
 
+interface TileDataProps {
+   color?: DataVisualizationColors,
+   disabled?: boolean,
+   id?: string,
+   onTap?:(
```

Comment:
Can you use AbstractEventHandler here?

```
AbstractEventHandler<
  | React.MouseEvent<HTMLDivElement>
  | React.KeyboardEvent<HTMLDivElement>
  | React.MouseEvent<HTMLAnchorElement>
  | React.KeyboardEvent<HTMLAnchorElement>,
  { selected: boolean, 
     id?: string 
    }
>

```


Also, this looks incorrect:
`React.MouseEvent<HTMLDivElement, MouseEvent`

Reply:
Okay sure

Reply:
Fixed, but is there any way to validate there's no syntax error or something?